charts/gha-runner-scale-set/templates/_helpers.tpl

```diff
@@ -75,12 +75,12 @@ app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
 
 {{- define "gha-runner-scale-set.dind-init-container" -}}
-{{- range $i, $val := .Values.template.spec.containers -}}
-{{- if eq $val.name "runner" -}}
+{{- range $i, $val := .Values.template.spec.containers }}
+{{- if eq $val.name "runner" }}
 image: {{ $val.image }}
 {{- if $val.imagePullSecrets }}
 imagePullSecrets:
```

Comment:
I think we have an error here
imagePullSecrets exist on a `pod.spec` level, not on the `pod.spec.containers` level

Reply:
Follow up: It should be safe just to remove it
If anyone wants to add it to the `template.spec`, it will be injected, so we should be fine :relaxed: